### ISSUE TITLE
Add support for providing password-store directory

### ISSUE THREAD

**Issue by erplsf** [synfinatic/aws-sso-cli#457]
**Is your feature request related to a problem? Please describe.**
I'm using to separate `password-store` directories on linux - one for personal and one for work related stuff encrypted with different GPG keys. When using `aws-vault` I was able to specify the directory to use for PASS backend via ENV variable: https://github.com/99designs/aws-vault/blob/106991e626c6938caf66289198ceb2a2142ca82d/USAGE.md#environment-variables (`AWS_VAULT_PASS_PASSWORD_STORE_DIR`). I'd like to see the same option, or something similar being present in `aws-sso-cli`.

**Describe the solution you'd like**
I can configure what path does `aws-sso-cli` use when it tries to use `pass` backend

**Describe alternatives you've considered**
I've tried implementing it myself in my fork, but I'm a bit of a newbie in Go, and not sure if my changes would work correctly / are maintainable.

**Additional context**

**Comment by synfinatic** [comment]
I assume you'd want to do this on a per-SSO instance? (`SSOConfig` key value in the config.yaml file). Or were you thinking of remembering to set an ENV variable each time???

**Comment by erplsf** [comment]
whatever you think is better/easier to implement. with `aws-vault` I was just setting ENV once in my `.zshrc` and be done with it. of course having it in `SSOConfig` would be better, because with ENV variable you need to remember to set it in non-interactive scripts manually.

**Comment by synfinatic** [comment]
Well the ENV variable thing seems kinda painful... like what happens if you forget to change the variable once?  Then you'll have work credentials in your personal secure store (or vise versa).   Honestly, I'm not sure how bad that would be?  Like what is the reason/threat model that has you separating the credentials?

**Comment by erplsf** [comment]
then let's have it in the static config file - I'm not against _any implementation_ that allows me to achieve what I want. the reason is simple, i believe - having separate password stores make sense because personal one is backed up by git, with personal GPG key, and the work related store is not backed up anywhere and protected by a different key, so if you have password to one you don't have all my passwords.

**Comment by synfinatic** [comment]
Is your threat model someone has installed a key logger on your system and is stealing your GPG passphrase or that you are choosing easily cracked passphrase?  Or a 3rd option???

Basically, I'm trying to balance the need for the additional complexity to add and support this feature long term.  If I do this for the GPG store, then I should do it for all secure stores right?  Right now it kinda seems that this attack can be mitigated by using a strong passphrase (I like [Diceware](https://en.wikipedia.org/wiki/Diceware) for my GPG passprhases) or is something that needs to be solved by using different systems (in the case of key logger).

Remember that the AWS API Keys that aws-sso stores in the secret store are time limited.  So unlike passwords they are only good for a limited amount of time (generally a few hours).

**Comment by erplsf** [comment]
Let's take a step back. I don't have a threat model, really, all I would like to have, is just *a way to configure the location where aws-sso (lib it uses) looks for `pass` repository/storage*, nothing more. I think that's the feature worth having, as not everyone wants/stores their `pass` repo at default `~/.password-store` location. I've managed in the end to do it in my fork so my usecase is satisfied for now: https://github.com/synfinatic/aws-sso-cli/commit/446c4d5a2a97cc42355c346345fcc1f877ee8bf6
Thank you for your time, and I don't want to sound like a jerk.

**Comment by synfinatic** [comment]
No worries and don't worry about sounding like a jerk. :)   I totally get what you want and I was trying to understand if there was a something I was missing as for why.  Historically I've tried to include every feature anyone has ever requested of me, but I've learned that can cause a lot of problems down the road due to the complexity.  It's always easier to add features than remove them later.  And if I add this then someone is going to want different OSX Keychains or Windows Credential Stores.

Anyways, I'm going to have to think about this.  I'll be honest, my inclination is to not include this feature because I don't see the security value/ROI and `aws-sso` arguably already has too many config options already.
